packages/grafana-ui/src/components/Table/RowsList.tsx

```diff
@@ -271,7 +271,7 @@ export const RowsList = (props: RowsListProps) => {
       const rowExpanded = nestedDataField && tableState.expanded[row.id];
 
       if (rowHighlightIndex !== undefined && row.index === rowHighlightIndex) {
-        style = { ...style, backgroundColor: theme.components.table.rowHoverBackground };
```

Comment:
I think we should consider updating the `theme.components.table.rowHoverBackground` color instead of pointing to `theme.colors.background.canvas`.

Reply:
Agreed.
Right now it's defined as `colors.emphasize(colors.background.primary, 0.03)`, should we just bump up the emphasize a bit? I'll post some screenshots in an hour or so after my next call

Reply:
Sounds good!

Reply:
Oh ok, so the theme.components.table.rowHoverBackground is also used on hover of the rows, not just when a row is "selected", so this will change a bit more then just the feature I was targeting, but I think that's ok.


Here's `colors.emphasize(colors.background.primary, 0.1)` with the "selected row", hovering looks a bit different, I'll get to that in a bit:
Light:
<img width="1705" alt="image" src="https://github.com/user-attachments/assets/5008fa0e-c1ae-4d8e-8981-d89d3ba573ec">
Dark:
<img width="1716" alt="image" src="https://github.com/user-attachments/assets/c29b352c-fb55-4fba-8587-676cd64a3c4d">

Here's 0.05
<img width="1698" alt="image" src="https://github.com/user-attachments/assets/fd8df710-d6eb-4562-9e11-fc0b84c43a33">
<img width="1702" alt="image" src="https://github.com/user-attachments/assets/8eab1f5a-448c-4571-a488-b9c8e1a3b46a">

Here's 0.075
<img width="1700" alt="image" src="https://github.com/user-attachments/assets/4203b1c1-81ae-4c5b-bb0c-5654dbcdff80">
<img width="1705" alt="image" src="https://github.com/user-attachments/assets/10ae25a9-13fe-48aa-b45f-44cb7b8d648d">


Hover seems to only change the cells in the row that you aren't hovering on: (with 0.075 value)
<img width="1706" alt="image" src="https://github.com/user-attachments/assets/d61605d3-b707-4804-a3ec-2a5f4e910db2">
<img width="1480" alt="image" src="https://github.com/user-attachments/assets/5722f8f5-7d0a-4271-9fe8-783e210dca5d">
<img width="876" alt="image" src="https://github.com/user-attachments/assets/b93d8da6-111e-420a-95c7-a0ea83491b32">
Don't think this is something we need to address in this PR, but I wanted to call it out

What do you think?

Reply:
Yeah, I noticed that too. Not sure it needs to be addressed here.

As for the emphasize value, I'll defer to @staton-hyse11's judgement!

Reply:
@staton-hyse11 is back in so should be able to take a look at this / unblock this soon :)

Reply:
Would using **action-selected** and **action-hover** work here?

Reply:
Works for me, quite a bit more noticeable then what we've got now.

<img width="1710" alt="image" src="https://github.com/user-attachments/assets/3711010e-2814-4ba2-a69c-4c26027b6f79">
<img width="1710" alt="image" src="https://github.com/user-attachments/assets/b36b947f-0e2a-4a7a-bb4c-f763cd0bdb02">

Not much differentiation between the two, but I think that's ok